src/bin/spin.rs

```diff
@@ -23,7 +24,10 @@ use spin_trigger::cli::TriggerExecutorCommand;
 async fn main() -> Result<(), Error> {
     tracing_subscriber::fmt()
         .with_writer(std::io::stderr)
-        .with_env_filter(tracing_subscriber::EnvFilter::from_default_env())
+        .with_env_filter(
+            tracing_subscriber::EnvFilter::from_default_env()
+                .add_directive("watchexec=off".parse()?),
```

Comment:
When I didn't do this `tracing::error!(..)`'s were making it through from watchexec.